tests/test_hyper_block.py

```diff
@@ -3,8 +3,8 @@
 from autokeras.auto.auto_model import *
 from autokeras import const
 
-from autokeras.hypermodel.hyper_block import XceptionBlock, RNNBlock
-from autokeras.hypermodel.hyper_head import ClassificationHead, RegressionHead
+from autokeras.hypermodel.hyper_block import *
+from autokeras.hypermodel.hyper_head import *
```

Comment:
This is test_hyper_block so we only do hyper_block import *, but not import * for other modules.

Reply:
Okay. got it